pkgs/by-name/js/jsoncons/package.nix

```diff
@@ -2,23 +2,25 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "jsoncons";
-  version = "0.178.0";
+  version = "1.1.0";
 
   src = fetchFromGitHub {
     owner = "danielaparker";
     repo = "jsoncons";
-    rev = "v${finalAttrs.version}";
-    hash = "sha256-tOHS3+xIbVHmzmNNO0xI4J313G5CcIze+DzUEcKTlcg=";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-3NG9Jal3+BdcFeWAIq3eFoV+1ywUCsWafe/WNu2VJU8=";
   };
 
   nativeBuildInputs = [ cmake ];
 
-  meta = with lib; {
+  env.NIX_CFLAGS_COMPILE = "-std=c++20 -Wno-error=deprecated-declarations";
```

Comment:
`-Wno-error` please.

Reply:
Why not `-Wno-error=deprecated-declarations`?

Reply:
Because the package shouldn't break on warnings, it makes updating more annoying.

Reply:
Good point.